src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/utils/ProcessorEnvelopeStatusChecker.java

```diff
@@ -32,14 +32,10 @@ public static String checkStatus(String fileName) {
         }
     }
 
-    public static boolean isEnvelopeCompleted(String fileName) {
+    public static boolean isEnvelopeCompletedWithCcdAction(String fileName) {
         JsonPath jsonPath = getZipFileStatusResponse(fileName).jsonPath();
-        if (jsonPath.getList("envelopes").isEmpty()
-            || !jsonPath.getString("envelopes[0].status").equals("COMPLETED")) {
-            return false;
-        } else {
-            return true;
-        }
+        return !jsonPath.getList("envelopes").isEmpty()
+            && !jsonPath.getString("envelopes[0].ccd_id").isBlank();
```

Comment:
can envelope be completed and not have ccd id set?

Reply:
seems yes.

Reply:
more likely test dont check correctly

Reply:
it is intermittent so it can not be test.